spec/factories/tasks.rb

```diff
@@ -27,5 +27,14 @@
     description { Faker::Lorem.paragraph }
 
     project
+
+    trait :with_files do
+      after(:build) do |user|
+        2.times do |n|
+          file = ["image1.jpg", "image2.jpg", "asana.png", "text.txt", "book.pdf"].sample
```

Comment:
wtf?
pls create a list of existing fixtures and assign appropriate content type that matches file content and extension